src/auth/oauth2client.ts

```diff
@@ -241,14 +245,20 @@ export class OAuth2Client extends AuthClient {
    * @param {Object=} opts optional options for overriding the given parameters.
    * @constructor
    */
+  constructor(options: OAuth2ClientOptions);
+  constructor(clientId?: string, clientSecret?: string, redirectUri?: string);
   constructor(
-      clientId?: string, clientSecret?: string, redirectUri?: string,
-      opts: AuthClientOpts = {}) {
```

Comment:
Making sure: the opts object was added in the `next` branch? The delta here looks semver major to me.

Reply:
Yikes, good catch.  Going to go add a test to make sure I don't break this.